po/de.po

```diff
@@ -1435,7 +1435,7 @@ msgid "Run research program for shooter"
 msgstr "Forschungsprogramm Shooterkanone"
 
 msgid "Run research program for target bot"
-msgstr ""
+msgstr "Forschungsprogramm Ziel-Roboter"
```

Comment:
Minor inconsistency: TargetBot is translated as *Mobile Zielscheibe* (object name/SatCom).

Reply:
Adapted.